example/src/Strada/Form.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from 'react-native';
+import React from 'react';
+import { BridgeComponent } from 'react-native-turbo';
+import { NavigationContext } from '@react-navigation/native';
+
+export default class Form extends BridgeComponent {
+  static name = 'form';
+  static contextType = NavigationContext;
+
+  constructor(props) {
+    super(props);
+  }
+
+  setHeaderButton(disabled: boolean) {
+    this.context.setOptions({
+      headerRight: () => (
+        <Button
+          onPress={() => this.replyTo('connect')}
+          title="Send"
+          disabled={disabled}
+        />
+      ),
+    });
+  }
+
+  onReceive(e) {
+    switch (e.event) {
```

Comment:
Consider changing `e` to `message` for clarity and be close to [Strada conventions](https://github.com/hotwired/strada-ios/blob/main/docs/BUILD-COMPONENTS.md?plain=1#L33-L34)

Reply:
Also consider defining and exporting the `StradaMessage` type